### PR TITLE
Bugfix/grab bag

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Fixes Borrow terms dropdown
- Fixes Lend fixed tables not populated initially
- Remove notification icon
- Fixes Fancy card page table
- Fixes Spacing on the TVL chart hiding number

### CODE REVIEW

File: packages/notionable/src/base-trade/sagas/tokens.ts [hunk 1]
```diff
@@ -9,21 +9,52 @@ import {
...
+  function getSelectedToken(
+    availableTokens: TokenDefinition[],
+    selectedToken: string | undefined,
+    category: Category,
+    tradeType?: TradeType | VaultTradeType | undefined,
+  ) {  
+    if (availableTokens.length === 1){
+      return availableTokens[0]
+    } else if(selectedToken === undefined) {
+      const defaultToken = getDefaultTokens(availableTokens, category, tradeType)
```
Comment (jeffywu) [inline packages/notionable/src/base-trade/sagas/tokens.ts:50]: Can't you just return directly here?

File: packages/notionable/src/base-trade/sagas/init-reset.ts [hunk 2]
```diff
@@ -11,18 +11,20 @@ export function resetOnTradeTypeChange(
     filterEmpty(),
     pairwise(),
     map(([prev, cur]) => {
+      // if (
```
Comment (jeffywu) [inline packages/notionable/src/base-trade/sagas/init-reset.ts:14]: Can you just delete this code so we don't get conflicts

File: packages/notionable/src/global/account/holdings.ts [hunk 3]
```diff
@@ -100,6 +100,7 @@ export function calculateHoldings(
       maturedTokenId,
       totalIncentiveEarnings,
       hasMatured: balance.hasMatured,
+      tokenType: undefined,
```
Comment (jeffywu) [inline packages/notionable/src/global/account/holdings.ts:103]: Why is this in here?

File: packages/shared/trade/src/leverage-terms/leverage-terms.tsx [hunk 4]
```diff
@@ -117,21 +124,30 @@ export const DefaultTerms = ({ context }: TermsProps) => {
...
 };
 
-export const ManageTerms = ({ context, linkString }: ManageTermsProps) => {
+export const ManageTerms = ({
+  context,
+  linkString,
+  isVault,
+}: ManageTermsProps) => {
   const history = useHistory();
+  const network = useSelectedCardNetwork();
```
Comment (jeffywu) [inline packages/shared/trade/src/leverage-terms/leverage-terms.tsx:133]: Here, use the `selectedNetwork` from the state

File: packages/shared/trade/src/leverage-terms/leverage-terms.tsx [hunk 5]
```diff
@@ -144,7 +160,7 @@ export const ManageTerms = ({ context, linkString }: ManageTermsProps) => {
       <TermsBox
         hasPosition={true}
         leverageRatio={leverageRatio}
-        borrowType={borrowType}
+        borrowType={isVault ? borrowTypeVaults : borrowTypeLeveragedLiquidity}
```
Comment (jeffywu) [inline packages/shared/trade/src/leverage-terms/leverage-terms.tsx:163]: Did you also remove the "Risk" text in the box?

File: packages/notionable/src/global/account/holdings.ts (hunk 3 above)
Comment (matthew-garrett) [reply packages/notionable/src/global/account/holdings.ts:103]: Fixes a type issue higher up. Doesn't hurt anything

**Pull request merged**